bl/evaluator.go

```diff
@@ -162,17 +158,12 @@ func (e *Evaluator) aggregateEvaluationResults(evaluationResults []cliClient.Eva
 
 			// file and rule not already exists in mapper
 			if _, exists := mapper[match.FileName][result.Rule.ID]; !exists {
-				filenames[match.FileName] = match.FileName
+				totalFailedCount++
```

Comment:
in case we have more than 1 occurrences in a file, this will cause more fails than expected

Reply:
no it won't :)